packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -1,6 +1,27 @@
 @import '~terra-mixins/lib/Mixins';
 
 :local {
+  // html input[type='button'].terra-Button selector used to
+  // prevent a WebKit bug destroys native `audio` and `video` controls
+  // in Android 4. More info:
+  // https://github.com/necolas/normalize.css/issues/20
+  // https://code.google.com/p/android/issues/detail?id=34301
+  // appearance: button; set to correct the inability to style
+  // clickable types in iOS and Safari.
+  // Appearance needs to be applied only to button/input elements, as
+  // applying to anchor element will cause render issue.
+  // More info:
+  // https://github.com/necolas/normalize.css/issues/566
+  // https://github.com/necolas/normalize.css/commit/1be44f3b2a79af5d463c7ea0264fee0e6fa464ca
+  // stylelint-disable selector-no-qualifying-type
+  button.button,
+  html input[type='button'].button,
+  input[type='reset'].button,
```

Comment:
We can remove the `type="reset"` and `type="submit"` selectors for this

Reply:
Resolved with https://github.com/cerner/terra-core/pull/2485/commits/a8bac0f3a5011a5bfe3cff8a50012ef539a822b0